engine/src/main/java/org/terasology/engine/world/chunks/pipeline/ChunkProcessingPipeline.java

```diff
@@ -301,8 +300,8 @@ public boolean isPositionProcessing(Vector3ic pos) {
      *
      * @return copy of processing positions
      */
-    public List<Vector3ic> getProcessingPosition() {
-        return new LinkedList<>(chunkProcessingInfoMap.keySet());
+    public Iterable<Vector3ic> getProcessingPosition() {
```

Comment:
@tolziplohu does this conflict with the changes you did on https://github.com/MovingBlocks/Terasology/compare/feat/order-chunks-on-demand?

Reply:
No, I actually didn't touch that method.